### ISSUE TITLE
LocateByLayer - Get geometry and display & PRINT it

### ISSUE THREAD

**Issue by josemvm** [3liz/lizmap-web-client#288]
hi,

i know that the lizmap printing system is based on qgis composer.
however, i think that the possibility of printing the highlight geometry it would be very useful!!!

**Comment by josemvm** [comment]
it's the same as https://github.com/3liz/lizmap-web-client/issues/464

**Comment by josemvm** [comment]
it's a regrettable loss
with no solution in sight... https://github.com/qgis/QGIS/pull/4027

**Comment by rldhont** [comment]
We probably get it back with QGIS 3
But the capapbility in QGIS Desktop to Print selection has been deactivated so we need to trick it for server.

**Comment by josemvm** [comment]
thanks @rldhont

**Comment by rldhont** [comment]
In the case of LocateByLayer, Lizmap does not use selection, so we could implement it with Redlining https://qgis.org/en/site/forusers/visualchangelog216/index.html#id99

**Comment by rldhont** [comment]
Documentation here: http://docs.qgis.org/testing/en/docs/user_manual/working_with_ogc/ogc_server_support.html#redlining

**Comment by josemvm** [comment]
it seems to be a good alternative

**Comment by josemvm** [comment]
related to https://github.com/3liz/lizmap-web-client/issues/288

**Comment by signedav** [comment]
Hi there - will there be a [redlining](https://docs.qgis.org/testing/en/docs/user_manual/working_with_ogc/server/services.html#redlining) implementation in Lizmap 3.3.0 for sure? This would be awesome.

**Comment by josemvm** [comment]
hi @signedav 

probably, you are talking about something like this :-)

![imagem](https://user-images.githubusercontent.com/10053874/49578698-41173d80-f942-11e8-91f8-113d8b0f272f.png)

export/import drawings and annotations to geojson file

**Comment by signedav** [comment]
Do you have more information about that @josemvm ?

**Comment by josemvm** [comment]
no, i have no idea. this is a p.mapper draw plugin implementation/dynamic layers

**Comment by nboisteault** [comment]
I close as you can print the geometry with an atlas configured.

**Comment by nboisteault** [comment]
I read to quickly. I reopen but I don't think this feature will be done with server printing. It will work when we'll develop client printing.
